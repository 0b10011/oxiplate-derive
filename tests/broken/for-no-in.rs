use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{%- for number numbers -%}{{ number }} + {{ number_to_add }} = {{ numb + number_to_add }}{% endfor %}"]
struct Data {
    numbers: Vec<i32>,
    number_to_add: i32,
}

fn main() {
    print!(
        "{}",
        Data {
            numbers: vec![19, 89],
            number_to_add: 4,
        }
    );
}
