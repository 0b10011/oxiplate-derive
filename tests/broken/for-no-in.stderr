error: Expected 'in'
 --> tests/broken/for-no-in.rs:4:37
  |
4 | #[oxiplate_inline = "{%- for number numbers -%}{{ number }} + {{ number_to_add }} = {{ numb + number_to_add }}{% endfor %}"]
  |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
